_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -3571,23 +3589,23 @@ mfxStatus ImplementationAvc::AsyncRoutine(mfxBitstream * bs)
                 if (!bRecoding && extOpt2.MaxSliceSize)
                 {
                     sts = CheckSliceSize(*task, bRecoding);
-                    if (sts != MFX_ERR_NONE)
-                        return Error(sts);
-                }
+                        if (sts != MFX_ERR_NONE)
```

Comment:
These are useless changes! Please, revert them. And actually you broke alignment.

Reply:
Sorry my bad, I did not realize I made this change. I will clean this up